app/dashboard/settings/profile/profileActions.ts

```
@@ -0,0 +1,21 @@
+"use server"
+
+import { revalidatePath } from "next/cache"
+
+import prisma from "@/lib/prisma"
+
+import { userSchema } from "./userSchema"
+
+export async function updateUser(data: any) {
+  debugger
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)